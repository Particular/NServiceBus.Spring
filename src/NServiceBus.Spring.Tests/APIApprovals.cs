using NServiceBus;
using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(SpringInternalType).Assembly);
        Approver.Verify(publicApi);
    }
}