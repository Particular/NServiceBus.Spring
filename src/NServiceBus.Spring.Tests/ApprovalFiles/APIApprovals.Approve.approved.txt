[assembly: System.CLSCompliantAttribute(true)]
[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute(@"NServiceBus.Spring.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100dde965e6172e019ac82c2639ffe494dd2e7dd16347c34762a05732b492e110f2e4e2e1b5ef2d85c848ccfb671ee20a47c8d1376276708dc30a90ff1121b647ba3b7259a6bc383b2034938ef0e275b58b920375ac605076178123693c6c4f1331661a62eba28c249386855637780e3ff5f23a6d854700eaa6803ef48907513b92")]
[assembly: System.Runtime.InteropServices.ComVisibleAttribute(false)]
[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
namespace NServiceBus
{
    [System.ObsoleteAttribute(@"Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the NServiceBus.Extensions.DependencyInjection library provides the ability to use any container that conforms to the Microsoft.Extensions.DependencyInjection container abstraction. Will be treated as an error from version 9.0.0. Will be removed in version 10.0.0.", false)]
    public class SpringBuilder : NServiceBus.Container.ContainerDefinition
    {
        public SpringBuilder() { }
        public override NServiceBus.ObjectBuilder.Common.IContainer CreateContainer(NServiceBus.Settings.ReadOnlySettings settings) { }
    }
    [System.ObsoleteAttribute(@"Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the NServiceBus.Extensions.DependencyInjection library provides the ability to use any container that conforms to the Microsoft.Extensions.DependencyInjection container abstraction. Will be treated as an error from version 9.0.0. Will be removed in version 10.0.0.", false)]
    public class static SpringExtensions
    {
        [System.CLSCompliantAttribute(false)]
        [System.ObsoleteAttribute(@"Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the NServiceBus.Extensions.DependencyInjection library provides the ability to use any container that conforms to the Microsoft.Extensions.DependencyInjection container abstraction. Will be treated as an error from version 9.0.0. Will be removed in version 10.0.0.", false)]
        public static void ExistingApplicationContext(this NServiceBus.Container.ContainerCustomizations customizations, Spring.Context.Support.GenericApplicationContext applicationContext) { }
    }
}